assets/main.js

```diff
@@ -252,26 +252,30 @@ function classLists() {
 
 
 var filtersCheck = document.querySelectorAll('.filter');
+function hideReports(reports) {
+  reports.forEach((item, i) => {
+    item.classList.remove('display');
+    item.style.display = "none";
+  });
+}
 function filters(){
- for(i=0; i<filtersCheck.length; i++) {
-   filtersCheck[i].addEventListener('click', function(){
-     document.querySelectorAll('.report-cover').forEach((item, i) => {
-       item.classList.remove('display');
-     });
-
+  var allReports = Array.from(document.querySelectorAll('.report-cover'))
+  hideReports(allReports)
+  console.log(allReports);
```

Comment:
I don't think this is intended on prod ✂️

Reply:
@raguiar9080 you mean the console.log?

Reply:
@nashthecoder yes